packages/tupaia-web-server/src/app/createApp.ts

```diff
@@ -107,7 +108,6 @@ export async function createApp(db: TupaiaDatabase = new TupaiaDatabase()) {
     .use('logout', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
     .use('projects', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
     .use('resendEmail', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
-    .use('signup', forwardRequest(WEB_CONFIG_API_URL, { authHandlerProvider }))
```

Comment:
This endpoint then forwards the request to central server's `user` endpoint to create the user, hence the error message doesn't get propagate to the front end. 

=> I don't see why we can't forward it directly to central server from tupaia-web-server...